apps/v3_api/test/routes_test.exs

```diff
@@ -6,8 +6,7 @@ defmodule V3Api.RoutesTest do
   @opts ["page[limit]": 1, sort: "long_name"]
 
   describe "all/1" do
-    %JsonApi{data: [%JsonApi.Item{} = route]} = Routes.all(@opts)
-    assert route.id == "80"
+    %JsonApi{data: [%JsonApi.Item{} | _]} = Routes.all(@opts)
```

Comment:
Given the `page[limit]=1`, shouldn't this have only been returning one item? Did the parameter stop working?

Reply:
no, I just missed it, will remove

Reply:
no, I just missed it, will remove